app/controllers/graphql_controller.rb

```diff
@@ -6,7 +6,53 @@ class GraphqlController < ApplicationController
   # but you'll have to authenticate your user separately
   # protect_from_forgery with: :null_session
 
-  skip_before_action :authenticate_user!, only: [:execute]
+  skip_before_action :authenticate_user!, only: %i[execute live_content]
+  before_action :set_cors_headers, only: [:live_content]
+
+  def live_content
+    execute_in_read_replica do
+      begin
+        encoded_base_path = Addressable::URI.encode(params[:base_path])
+
+        schema_name = Edition.live.find_by(base_path: encoded_base_path)&.schema_name
```

Comment:
At some point, we're going to need to port the [non-exact routing logic from content-store](https://github.com/alphagov/content-store/blob/main/app/lib/find_by_path.rb#L15-L81) over I think. That probably warrants a separate ticket though.

Reply:
Indeed.  I noted this in the commit message, but I'll add a note to the "make a batch of 10 content schemas ready" card as a prerequisite if we include any formats that use non-exact routes.